### DISCUSSION TITLE
Is there a way to customize the text browsers display?

### DISCUSSION THREAD

**Discussion by liamhalemccarty** [MasterKale/SimpleWebAuthn#494] [Troubleshooting]
I'm curious if there's a way to customize the text that browsers display to users when they register/use passkeys. I've found that the text makes sense for some use cases but not so much for others. 

For example, Safari desktop displays this:

<img width="491" alt="image" src="https://github.com/MasterKale/SimpleWebAuthn/assets/37675947/2ce649c1-52ed-4a8a-a371-9bed69c46b25">

I haven't seen a way to customize this text. Does anyone know of a way to do this?

**Comment by hwhmeikle** [comment]
AFAIK there is no way to customise this.
